### PR TITLE
TimeSpans in Service Fabric config

### PR DESCRIPTION
**Pull request by Kahbazi**
Fixes https://github.com/microsoft/reverse-proxy/issues/589

### CODE REVIEW

File: src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs [hunk 1]
```diff
@@ -196,27 +196,23 @@ internal static Cluster BuildCluster(Uri serviceName, Dictionary<string, string>
                     Active = new ActiveHealthCheckOptions
                     {
                         Enabled = GetLabel(labels, "YARP.Backend.HealthCheck.Active.Enabled", false),
-                        Interval = ToNullableTimeSpan(GetLabel<double?>(labels, "YARP.Backend.HealthCheck.Active.Interval", null)),
-                        Timeout = ToNullableTimeSpan(GetLabel<double?>(labels, "YARP.Backend.HealthCheck.Active.Timeout", null)),
+                        Interval = GetLabel<TimeSpan?>(labels, "YARP.Backend.HealthCheck.Active.Interval", null),
```
Comment (alnikola) [inline src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs:199]: Could you please add a test verifying handling of `null` and an empty string by `GetLabel<TimeSpan?>`? I'm a bit suspicious that it works as expected.

**Review by alnikola — COMMENTED** [review]
Please add tests verifying missing values handling.

File: src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs (hunk 1 above)
Comment (Kahbazi) [reply src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs:199]: @alnikola I've added tests for null and empty string.

**Review by alnikola — APPROVED** [review]
LGTM

**Comment by alnikola** [comment]
Thanks!
